draft-ietf-scitt-architecture.md

```diff
@@ -558,7 +558,7 @@ The subsequent sections describe the main concepts, namely Transparency Service,
 Transparency Services MUST feature a Verifiable Data Structure.
 The Verifiable Data Structure records registered Signed Statements and supports the production of Receipts.
 
-All Transparency Services MUST expose APIs ({{-SCRAPI}} for the Registration of Signed Statements and issuance of Receipts.
+All Transparency Services MUST expose a minimally conformant API ({{-SCRAPI}} for the Registration of Signed Statements and issuance of Receipts.
```

Comment:
```suggestion
All Transparency Services MUST expose the minimally conformant APIs, defined in ({{-SCRAPI}} for the Registration of Signed Statements and issuance of Receipts.
```

Reply:
+1, this makes the MUST clearer for me

Reply:
I sorta wonder why we are making SCRAPI MTI, but I don't object to these specific changes